controllers/operands/ssp_test.go

```diff
@@ -748,40 +676,36 @@ var _ = Describe("SSP Operands", func() {
 					hco := commontestutils.NewHco()
 					hco.Spec.EnableCommonBootImageImport = ptr.To(true)
 
-					disabledImage1 := image1.DeepCopy()
-					disableDict(disabledImage1)
-					enabledImage2 := image2.DeepCopy()
-					enableDict(enabledImage2)
+					disabledImage1, _ := makeDICT(1, true)
+					disableDict(&disabledImage1)
+					enabledImage2, expectedStatus2 := makeDICT(2, true)
+					enableDict(&enabledImage2)
+					expectedStatus2.Status.Modified = true
 
-					hco.Spec.DataImportCronTemplates = []hcov1beta1.DataImportCronTemplate{*disabledImage1, *enabledImage2, image3, image4}
+					hco.Spec.DataImportCronTemplates = []hcov1beta1.DataImportCronTemplate{disabledImage1, enabledImage2, image3, image4}
 					goldenImageList, err := getDataImportCronTemplates(hco)
 					Expect(err).ToNot(HaveOccurred())
 					Expect(goldenImageList).To(HaveLen(3))
 					Expect(goldenImageList).To(HaveCap(4))
 
-					statusImage2Enabled := statusImage2.DeepCopy()
-					statusImage2Enabled.Status.Modified = true
-
-					Expect(goldenImageList).To(ContainElements(*statusImage2Enabled, statusImage3, statusImage4))
+					Expect(goldenImageList).To(ContainElements(expectedStatus2, statusImage3, statusImage4))
 				})
 
 				It("should not add user DIC template if it is disabled", func() {
 					dataImportCronTemplateHardCodedMap = nil
 					hco := commontestutils.NewHco()
 					hco.Spec.EnableCommonBootImageImport = ptr.To(true)
 
-					disabledUserImage := image1.DeepCopy()
-					disableDict(disabledUserImage)
-					enabledUserImage := image2.DeepCopy()
-					enableDict(enabledUserImage)
+					disableDict(&image1)
+					enableDict(&image2)
 
-					hco.Spec.DataImportCronTemplates = []hcov1beta1.DataImportCronTemplate{*disabledUserImage, *enabledUserImage}
+					hco.Spec.DataImportCronTemplates = []hcov1beta1.DataImportCronTemplate{image1, image2}
```

Comment:
didn't `disabledUserImage`, `enabledUserImage` and `image3Modified` variable names had a special meaning before?

Reply:
It's just that now, when the  image1-4 variables are re-created at `BeforeEach`, the custom images (created by DeepCopy and then modified), are not needed anymore.